assets/model_monitoring/components/src/model_monitor_output_metrics/entities/signals/generation_safety_quality_signal.py

```diff
@@ -13,6 +13,21 @@
 from shared_utilities.run_metrics_utils import get_or_create_run_id, publish_metric
```

Comment:
Let's use the metric outputter instead! This will be in line with Output v2
https://github.com/Azure/azureml-assets/tree/main/assets/model_monitoring/components/model_monitor/model_monitor_metric_outputter